R/derive_merged.R

```diff
@@ -0,0 +1,848 @@
+#' Add New Variable(s) to the Input Dataset Based on Variables from Another
+#' Dataset
+#'
+#' Add new variable(s) to the input dataset based on variables from another
+#' dataset. The observations to merge can be selected by a condition
+#' (`filter_add` argument) and/or selecting the first or last observation for
+#' each by group (`order` and `mode` argument).
+#'
+#' @param dataset Input dataset
+#'
+#'   The variables specified by the `by_vars` parameter are expected.
+#'
+#' @param dataset_add Additional dataset
+#'
+#'   The variables specified by the `by_vars`, the `new_vars`, and the `order`
+#'   parameter are expected.
+#'
+#' @param order Sort order
+#'
+#'   If the parameter is set to a non-null value, for each by group the first or
+#'   last observation from the additional dataset is selected with respect to the
+#'   specified order.
+#'
+#'   *Default*: `NULL`
+#'
+#'   *Permitted Values*: list of variables or `desc(<variable>)` function calls
+#'   created by `vars()`, e.g., `vars(ADT, desc(AVAL)` or `NULL`
```

Comment:
there is a missing parethesis after AVAL)

Reply:
Thanks, fixed